.github/workflows/deploy-storybook.yml

```
@@ -0,0 +1,36 @@
+
+on:
+  push:
+    branches: [develop]
+    paths:
+      - 'packages/design-system/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)